src/test/java/nextstep/subway/acceptance/LineAcceptanceTest.java

```diff
@@ -0,0 +1,122 @@
+package nextstep.subway.acceptance;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static nextstep.subway.acceptance.LineSteps.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("지하철 노선 관리 기능")
+class LineAcceptanceTest extends AcceptanceTest {
+    /**
+     * When 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+    @DisplayName("지하철 노선 생성")
+    @Test
+    void createLine() {
+        // when
+        ExtractableResponse<Response> response = 지하철_노선_생성_요청("2호선", "green");
```

Comment:
```suggestion
        var response = 지하철_노선_생성_요청("2호선", "green");
```

타입 추론인 var를 사용하는 것은 어떨까요~?

Reply:
제가 추론을 써본적이 없는데 이번기회에 한번 써보겠습니다 😎 반영하겠습니다.